variants/devise/template.rb

```diff
@@ -68,6 +71,28 @@ def print_header(msg)
           /  # config.secret_key = '.+'/,
           "  # config.secret_key = 'do_not_put_secrets_in_source_control_please'"
 
+gsub_file "config/initializers/devise.rb",
+          /  # config.lock_strategy = .+/,
+          "  config.lock_strategy = :failed_attempts"
+
+gsub_file "config/initializers/devise.rb",
+          /  # config.unlock_strategy = .+/,
+          "  config.lock_strategy = :email"
+
+gsub_file "config/initializers/devise.rb",
+          /  # config.maximum_attempts = .+/,
+          <<-EO_CHUNK
+  #
```

Comment:
Deliberately using `<<-` here to let us insert leading whitespace to match the rest of the file.